usagov/vote-gov-nvrf-app
# Cypress update for NVRF

## Jira ticket
Multiple Tickets Covered within this PR:
[Vote-511](https://cm-jira.usa.gov/browse/VOTE-511)
[Vote-514](https://cm-jira.usa.gov/browse/VOTE-514)
[Vote-512](https://cm-jira.usa.gov/browse/VOTE-512)
[Vote-508](https://cm-jira.usa.gov/browse/VOTE-508)

## Description

This PR will update and add testing for the NVRF.

Update - Testing has been updated to cover the new workflow post-redesign 
Addition - Added testing for the back button within the form, validate that fields within the form are correct, validate that the edit button on the confirmation page takes the user to the right page and validate that the info on the confirmation page contains the correct information 

## Deployment and testing

### Pre-deploy

n/a

### Post-deploy

n/a

### QA/Test

1. Pull down branch 
2. run `npm run start` and then split terminal
3. within the new terminal window cd into `testing`
4. run `npm run cy:test` and verify that tests are passing 
5. run `npm run cy:open` and click through tests and verify that tests are passing 

## Checklist for the Developer




## Checklist for the Peer Reviewers
